meta-mender-core/recipes-mender/mender-client/mender-client-cpp_git.bb

```diff
@@ -0,0 +1,14 @@
+require mender-client-cpp.inc
+
+LICENSE = "Apache-2.0 & BSD-3-Clause"
+
+LIC_FILES_CHKSUM = " \
+    file://LICENSE;md5=4cd0c347af5bce5ccf3b3d5439a2ea87 \
+    file://vendor/googletest/LICENSE;md5=cbbd27594afd089daa160d3a16dd515a \
+    file://vendor/json/LICENSE.MIT;md5=f969127d7b7ed0a8a63c2bbeae002588 \
+    file://vendor/json/docs/mkdocs/docs/home/license.md;md5=970ea048f6ea7d04eeb3ba3ef9ebca40 \
+"
+
+SRCREV = "${AUTOREV}"
```

Comment:
Shouldn't this be the `mender_autorev_if_git_version` variant from the original recipe?

Reply:
Yup

Reply:
I will leave this be until the last iteration

Reply:
Done